core/web/log_controller.go

```diff
@@ -0,0 +1,101 @@
+package web
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/gin-gonic/gin"
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/chainlink/core/services/chainlink"
+	"github.com/smartcontractkit/chainlink/core/web/presenters"
+	"go.uber.org/zap/zapcore"
+)
+
+// LogController manages the logger config
+type LogController struct {
+	App chainlink.Application
+}
+
+type LoglevelPatchRequest struct {
+	LogLevel string `json:"logLevel"`
+	LogSql   string `json:"logSql"`
+}
+
+func getLogLevelFromStr(logLevel string) (zapcore.Level, error) {
```

Comment:
`zap` has a function to help you with this

https://github.com/uber-go/zap/blob/404189cf44aea95b0cd9bddcb0242dd4cf88c510/zapcore/level.go#L116